lib/bundlex/loader.ex

```diff
@@ -140,11 +140,15 @@ defmodule Bundlex.Loader do
         :ok
       else
         {:error, {reason, text}} ->
-          raise """
+          require Logger
+
+          Logger.error("""
```

Comment:
I'm quite sure. I changed it from `raise` because previously we'd get both error and warning, both via Logger. I think `Bundlex.Output.warn` would work too, I'm not sure about other `Bundlex.Output` functions, so I'd not use it in runtime

Reply:
Ok, so I checked and in the compilation the bundlex compilation is called first and there we do `Application.ensure_all_started` https://github.com/membraneframework/bundlex/blob/master/lib/mix/tasks/compile.bundlex.ex#L22 and because we have `:logger` in `extra_applications` in `mix.exs`, the logger should be started at that point. So, it seems that the logger is started before any C code is even compiled.

Reply:
If so, is there any reason to call `IO.warn/1` in `Bundlex.Output.warn/1` or `Mix.shell().info/1` in `Bundlex.Output.info/1`?

Reply:
I don't know why we do this, probably we could use Logger as well ¯\\_(ツ)\_/¯